### ISSUE TITLE
Customize autoform array display template

### ISSUE THREAD

**Issue by jccgadam** [aldeed/meteor-autoform#989]
Hi: 
I am working with autoform and have a hard time try to customize array inserting template.
If I use {{> afQuickField name='POs'}} POs is defined in simpleschema as an array with 4 fields.
This way works fine, showing the form with add more element button shown and every thing which is really COOL!
But each field of POs takes the whole line like:
field 1:_____________________________________
field 2:_____________________________________
field 3:_____________________________________
field 4:_____________________________________

But I am trying to make four fields in same line like:
field 1:    field 2:    field 3:    field 4:
_____    ______   _____    _____

In summary I am trying to customize this display template 
I have tried:
 {{#afEachArrayItem name='POs'}}
        {{> afFieldInput name='this.current.field1'}}  
        {{> afFieldInput name='this.current.field2l'}}
    {{/afEachArrayItem}}
but do not work
Can you help me on this problem ?
thanks a lot

**Comment by andmitsch** [comment]
Could you post your schema code?

**Comment by jccgadam** [comment]
Thanks a lot for the reply.
Here is my schema. 
// eachPOSchema  is one po record and POs schema stores POs as array 
// 
Schemas.eachPOSchema=new SimpleSchema({
poNum:{
  type:String,
  label:"P/O Num",
  },
model:{
  type:String,
  label:"Model"
},
singlePrice:{
  type:Number,
  decimal:true,
  label:"Price"
},
quantity:{
  type:Number,
  label:"Quantity"
}
 })

Schemas.POsSchema=new SimpleSchema({
POs:{
	type:[Schemas.eachPOSchema],
	minCount:2,
},
 totalPrice:{
  type:Number,
  decimal:true,
   },
 attachCI:{
  type:Boolean,
 },
 attachPC:{
  type:Boolean,
 },
 attachQA:{
  type:Boolean,
 },
 attachISF:{
  type:Boolean,
 },
 attachBOL:{
  type:Boolean,
 },
 estimateShipTime:{
  type:Date,
 }
})

**Comment by jccgadam** [comment]
//besides I tried quickForm which offers all the functionalists I need but I cannot customize the layout. 
{{#quickForm collection="pos" id="posForm" tyep="insert"}}
{{/quickForm}}

// for experiencing I also tried afEachArrayItem, but not getting the output that I expected
{{#afEachArrayItem name='POs'}}
        {{> afFieldInput name='this.current.poNum'}}  
        {{> afFieldInput name='this.current.model'}}
    {{/afEachArrayItem}}

**Comment by andmitsch** [comment]
`pos`is a collection in `Schemas` ? (Not necessary for this case, but I want to learn this way of using af)

You can customize the quickForm layout by including additional classes and/or using a different template, or even your own custom template where you use your own markup.

```html
{{> quickForm 
collection="pos"
id="posForm" 
template="bootstrap3-inline" 
class="inline-form-group" //this class gets attached to the div.form-group
label-class=""
input-col-class="" 
}}`

**Comment by jccgadam** [comment]
Thanks a lot for the reply. I will try this.
